Makefile.pre.in

```diff
@@ -1431,6 +1431,9 @@ libinstall:	build_all $(srcdir)/Modules/xxmodule.c
 			*CVS) ;; \
 			*.py[co]) ;; \
 			*.orig) ;; \
+			# bpo-37468: Don't install distutils/command/wininst-*.exe files used \
+			# by distutils bdist_wininst: bdist_wininst only works on Windows. \
+			*wininst-*.exe) ;; \
```

Comment:
Adding comments here breaks `make libinstall` and hence `make install`, at least for some types of make.  Fixed by bpo-37653 / #14911.